10_datastructures_lab/01.py

```diff
@@ -1,19 +1,8 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-
-When valid credentials are entered print:
-    Welcome Master
-
-And when invalid credentials are entered print:
-    INTRUDER ALERT
-
-and exit the program with a non-zero exit code
-"""
-
+"""get name and password and check if it correct"""
+import os,sys
+nep = {'apple' : "red" , 'lettuce' : "green" , 'lemon' : "yellow" , 'orange' : "orange"}
+for key in nep:

```

Comment:
In addition users don't normally enter their passwords as command line arguments, because anyone can easily see it (by showing for example all processes running on the machine).

It's better to read this data using `raw_input`